### PR TITLE
Update cookiecutter to integration standards

### PR DESCRIPTION
**Pull request by erikzaadi**
* Proper `.env.example` files
* Update vscode launch configuration with new generated integration
* Add initial static examples to see the integration quickly in Port,
  including example blueprints and mappings
* Add unit test stub

# Description

Update the cookiecutter template for new integrations with integration standards.

In addition, make it simpler to start developing an integration with a "working" sample out of the box.

## Type of change

Please leave one option from the following and delete the rest:

- [x] Non-breaking change (fix of existing functionality that will not change current behavior)

### CODE REVIEW

**Review by matan84 — CHANGES_REQUESTED** [review]
Left a few comments

File: port_ocean/cli/cookiecutter/{{cookiecutter.integration_slug}}/changelog/.gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
+*
```
[the remark this answers is not in the document]
Comment (erikzaadi) [reply port_ocean/cli/cookiecutter/{{cookiecutter.integration_slug}}/changelog/.gitignore:1]: Also added in all integrations, prevents `towncrier` leftovers

File: port_ocean/config/settings.py [hunk 2]
```diff
@@ -114,3 +115,5 @@ def validate_runtime(cls, runtime: Runtime) -> Runtime:
                 raise ValueError("This integration can't be ran as Saas")
 
         return runtime
+
+        return runtime
```
[the remark this answers is not in the document]
Comment (erikzaadi) [reply port_ocean/config/settings.py:119]: WHAT IN THE NAME OF ODINS BEARD

**Review by matan84 — APPROVED** [review]
LGTM

**Pull request merged**